releases/release-1.31/release-notes/maps/pr-122234-map.yaml

```
@@ -0,0 +1,18 @@
+pr: 122234
+releasenote:
+  text: "The scheduler starts to use QueueingHint registered for Pod/Updated event
```

```suggestion
  text: "Added support to the scheduler to start using QueueingHint registered for Pod/Updated event
```
https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653157158